app/views/partials/_recipe.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class='recipe_name_delete'>
+    <a href="/recipes/<%= recipe.id %>">
+    <%= recipe.name %> <%= recipe.id %>
+    </a>  
+    
+    <% if can? :destroy %>
+        <a href="<%= recipe_path( recipe) %>"
+        data-turbo-method="delete"
+        data-turbo-confirm="Are you sure?"
+        data-method="delete      
+        ">DELETE RECIPE</a>
+    <% end %>
+     
+</div>
+
+<div class='recipe_description'>
+    <%= recipe.description %>
+</div>
+
```

Comment:
- [ ] i recieve this error when i visit recipe page after creating a private recipe , kindly fix it 🙏 

![image](https://github.com/Tshobohwa/recipe-app/assets/89282221/976b1b0d-f1a8-4578-9c4b-71fee11776e3)